### ISSUE TITLE
Double or triple the wormhole spamming rate by disabling client-side cooldown check

### ISSUE THREAD

**Issue by kaetemi** [wchill/steamSummerMinigame#254]
By disabling client-side check on Wormhole cooldown, you avoid the 1second overhead for waiting for the cooldown response from the server. (The server can check against a more up-to-date value.)

Tested on room 44888 this allows literally 1 Wormhole EVERY SECOND during wormhole rounds (thanks to all the Like New).

(Limited the flood mode to bosses >50% hp in my tests to avoid wasting unnecessarily in case of lag.)

**Comment by kaetemi** [comment]
In action in MonsterGUI: http://imgur.com/amjBGMv

**Comment by wchill** [comment]
Got it, I'll make sure to integrate this. Thanks.

**Comment by ryantheleach** [comment]
Is it possible to use all of your WH in a single tick by stuttering the usage between wormholes and LikeNew?

**Comment by kaetemi** [comment]
Multiple WH in a single tick has no effect. Have not tried a message with alternating WH and LN yet though.

**Comment by kaetemi** [comment]
However, I can confirm that it's possible to send more than one WH per second if you do multiple POST requests spread out (also thanks to the Like New being processed often enough).

Currently pushing in 3 wormholes per second...

**Comment by blole** [comment]
Actually, the tooltip for Like New says: "While active, clears the cooldown on every ability [...] Duration 0:05". Which would mean that using one Like New would allow you to spam wormholes for the next 5 seconds, my experiments seem to confirm this.

**Comment by kaetemi** [comment]
blole: Others experiments say only one cooldown reset on activation.

**Comment by blole** [comment]
Aha, well here's my experiment

run in console:
```setInterval(function(){g_Minigame.m_CurrentScene.m_rgAbilityQueue.push({'ability': 26})}, 500);```
and then manually press Like New

This uses up 2-6 of my wormholes.

**Comment by ryantheleach** [comment]
In a game without anyone else using like new?

**Comment by blole** [comment]
Yes.

**Comment by blole** [comment]
Okay, with the following code in an empty room I managed to use up ~950 Wormholes by myself in 5 minutes.
```javascript
setInterval(function(){
  g_Minigame.m_CurrentScene.m_rgAbilityQueue.push({'ability': 26}); //wormhole
  g_Minigame.m_CurrentScene.m_nLastTick = 0;
  g_Minigame.m_CurrentScene.Tick();
}, 10);
setInterval(function(){
  g_Minigame.m_CurrentScene.m_rgAbilityQueue.push({'ability': 27}); //like new
}, 2000);
```
This would mean ~3.1 wormholes/second, and the interval of 10 ms is probably unnecessarily low. I also don't know how well increasing the tick rate would work on a busy server.

**Comment by ryantheleach** [comment]
Yeah had wormholes working in slave windows in tabs using stock wchill in 45810.

**Comment by g1real** [comment]
Does this still work? I tested it myself earlier and it didn't.
